docs/howto/import.md

```diff
@@ -22,6 +22,22 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
 
 ### Prerequisites
 
+#### User Permissions
+
+The lakeFS user running the import must have the following permissions:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit` and 
+`fs:ImportFromStorage`. The first 3 permissions are available by default to 
+users in the default Developers group ([RBAC](../reference/rbac.md)) or the 
+Writers group ([ACL](../reference/access-control-lists.md)). The last 
+permission enables the user to import data from any location of the storage 
+provider that lakeFS has access to. Thus, it's only available to users in the 
+default SuperUsers group ([RBAC](../reference/rbac.md)) or the Super 
```

Comment:
I think that we should rephrase to make it more procedural.  Perhaps something like:

> ... only available to users in group Supers (on ACL) or SuperUsers (on RBAC)

?  And similarly for all other alternatives.

Reply:
done.